packages/authentication/src/definitions.ts

```diff
@@ -591,6 +597,24 @@ export interface UpdatePasswordOptions {
   newPassword: string;
 }
 
+/**
+ * @since 1.4.0
+ */
+export interface UpdateProfileOptions {
+  /**
+   * The user's display name.
+   *
+   * @since 1.4.0
+   */
+  displayName?: string | null;
+  /**
+   * The user's photo URL.
+   *
+   * @since 1.4.0
+   */
+  photoURL?: string | null;
```

Comment:
```suggestion
  photoUrl?: string | null;
```

I know it's called `photoURL` in the Firebase docs, but I already have it in use as `photoUrl` (see [User](https://github.com/capawesome-team/capacitor-firebase/tree/main/packages/authentication#user)).